src/api/app/assets/javascripts/webui2/repositories.js

```diff
@@ -1,22 +1,85 @@
 function setSpinnersForFlags() { // jshint ignore:line
-  $(document).on('click','.popover_flag_action', function() {
-    var flag = $(this).data('flag-id');
-    var icon = $('div[id="' + flag + '"] a');
-    icon.addClass('d-none');
-    icon.next().removeClass('d-none');
-    $('.popover').popover('hide');
-  });
+    $(document).on('click', '.popover_flag_action', function() {
+        var flag = $(this).data('flag-id');
```

Comment:
this indenting was done by atom's beautifier. Does obs have a indentation policy on javascript?

Reply:
I think it is 2 spaces :thinking: